packages/larva-css/src/02-algorithms/a-hover-effect/a-hover-effect.common.async.scss

```diff
@@ -12,4 +12,14 @@
 	--a-hover-effect-property: opacity;
 
 	opacity: 0.5;
-}
\ No newline at end of file
+}
+
+.lrv-a-hover-effect--underline {
+	transform: translate3d( 0px, 0px, 0px ); // stylelint-disable-line unit-blacklist
+}
+
+.lrv-a-hover-effect--underline:hover {
+	--a-hover-effect-property: text-decoration;
+
+	text-decoration: underline;
```

Comment:
We might want to use `border-bottom` with `currentColor` here instead so that design can have more control. Thoughts, @sarahsmo?

Reply:
ah yeah! good thinking!